astro/src/content/docs/get-started/start-here/index.mdx

```diff
@@ -0,0 +1,205 @@
+
+To understand how the JavaScript application integrates with FusionAuth, take a quick look at some of the code from the repository.
+
+We craft a URL that includes the `clientId` from the configuration you reviewed above, as well as some other parameters. This is where the user goes to log in with FusionAuth. TODO need to only show the `redirect(302, `fusionAuthURL/oauth2/authorize` line
+
+<RemoteCode url="https://raw.githubusercontent.com/synedra/fusionauth-example-express-start-here/main/app/src/index.ts" lang="javascript" tags="login" />
+
+Once the user authenticates, the code at the configured redirect URL is called. This code sends parameters from the login event (the authorization code), the client identifier (clientId) and other information to get a valid token from FusionAuth. When the app receives this token, the user has fully been authenticated. TODO need to only show the try block here
+
+<RemoteCode url="https://raw.githubusercontent.com/synedra/fusionauth-example-express-start-here/main/app/src/index.ts" lang="javascript" tags="oauth-redirect" />
+
+Once you have the token, what should the app do with it? One option is to store it as a secure cookie. We do that here. TODO show cookie saving
+
+We now can protect app functionality. For the account page, which requires authorization, the user's identity is checked and the browser is redirected to the main page if there is no valid token.
+
+<RemoteCode url="https://raw.githubusercontent.com/synedra/fusionauth-example-express-start-here/main/app/src/index.ts" lang="javascript" tags="account" />
+
+### Open the Application URL in your Browser and Login
+
+In your shell, type the following to start up the application:
+
+<Code code="cd app && npm install && npm run dev" />
+
+Point your browser to the application:
+
+GitPod:
+<Code code="gp preview `gp url 8080` --external" />
+
+Local:
+<Code code="open http://localhost:8080" />
+
+Now, login to the application in your browser using richard@example.com and 'password' to see the integration working.
+
+### What Did You Just Do?
+
+You looked at some FusionAuth configuration, started a JavaScript application and checked out some cod, and logged in to the application using FusionAuth.
+
+## Add Functionality
+
+By delegating to FusionAuth, your application gets access to authentication and authorization functionality, such as magic link login, multi-factor authentication, social sign-on, self-service registration and more. This is where a lot of the value comes from.
+
+Enabling your users to log in using magic links can reduce friction and user frustration. Let's enable this on the application you just set up.
+
+### Login With Magic Links
+
+To add a `Login With Magic Links` button, you usually have to configure an email server and then FusionAuth. But for this guide, we've already set up the email server. It is a locally running mock email server, and you can visit your inbox at [http://localhost:1080](http://localhost:1080).
```

Comment:
```suggestion
To add a `Login With Magic Links` button, you usually have to get access to an email server and configure FusionAuth to use it. But for this guide, we've already set up the email server. It is a locally running mock email server, and you can visit your inbox at [http://localhost:1080](http://localhost:1080).
```

Reply:
Is that the right URL for gitpod? Probably not, right? Do we need to have two sets of instructions here?

Reply:
@synedra  any thoughts here?